app/models/ems_event.rb

```diff
@@ -146,6 +146,34 @@ def self.process_vm_in_event!(event, options = {})
         event[:vm_name] ||= vm.name
       end
     end
+
+    event_type = event[:event_type]
+    # both event types are Rhev specific
+    if (event_type == 'USER_ADD_VM' || event_type == 'USER_ADD_VM_FINISHED_SUCCESS') && event[:vm_or_template_id].nil?
```

Comment:
> So it does only work if something already exists on our side in the path, like the host.
Do you mean it only works if they exist in the database already?  A targeted refresh should work even if none of the associated inventory was in the database yet AFAIK

It shouldn't be _that_ bad (easy for me to say) to try to use the existing `RefreshParser` and just get the targeted inventory in `collect_inventory_for_targets`

(can't wait for :skull: refresh either)